gui/src/widgets/popupwidget.cpp

```diff
@@ -0,0 +1,95 @@
+#include "widgets/popupwidget.h"
+#include "stylehelper.h"
+#include "ui_popupwidget.h"
+
+
+using namespace scopy::gui;
+
+PopupWidget::PopupWidget(QWidget* parent) :
+	ui(new Ui::PopupWidget),
+	QWidget(parent)
+{
+	ui->setupUi(this);
+
+	StyleHelper::initColorMap();
+	StyleHelper::BlueButton(ui->btnContinue, "btnContinue");
+	StyleHelper::BlueButton(ui->btnExit, "btnExit");
+	StyleHelper::OverlayMenu(this, "popupOverlay");
+	ui->title->setStyleSheet("background-color: #101014; border: 1px solid #070708"); // color match the description QTextBrowser
```

Comment:
This belongs to a stylehelper. Create a stylehelper with an appropriate name.

Reply:
Should the color codes also have an entry in the StyleHelper colorMap, or is it safe to assume that because the codes are not used anywhere else the string can be left "as is"?

Reply:
Yes they should be a part of the colormap with apropriate naming.